infrastructure/v1/servicemesh_types.go

```diff
@@ -5,7 +5,7 @@ import operatorv1 "github.com/openshift/api/operator/v1"
 // ServiceMeshSpec configures Service Mesh.
 type ServiceMeshSpec struct {
 	// +kubebuilder:validation:Enum=Managed;Removed
-	// +kubebuilder:default=Managed
+	// +kubebuilder:default=Removed
```

Comment:
will this lead to CreateDefaultDSCI() use Removed for ServiceMesh?

Reply:
1. Yep, hence updated the CreateDefaultDSCI as well https://github.com/red-hat-data-services/rhods-operator/pull/149/commits/a386b6100872a395e22f480b10d6f781c45510c6